doc/data/glossary/glossary.yml

```diff
@@ -0,0 +1,182 @@
+# Glossary of terms
+
+# Formatted as:
+
+# <Term>:
+#  description: <Description>
+#  link: <Link>
+#  ref: <Ref>
+#  abbr: <Abbreviation>
+#  alt:
+#    - <Alt term 1>
+#    - <Alt term 2>
+
+Gateway:
+  description: Gateways form the bridge between devices and Network Servers. Devices use low power networks like LoRaWAN to connect to the Gateway, while the Gateway uses high bandwidth networks like WiFi, Ethernet or Cellular to connect to The Things Network. // antennas that receive broadcasts from End Devices and send data back to End Devices.
+  link: https://www.thethingsnetwork.org/docs/gateways/
+
+Gateway ID:
+  description: A unique, human-readable identifier for your gateway.
+
+Gateway EUI:
+  description: A 64 bit extended unique identifier for your gateway. This is programmed by the manufacturer and should not be changed. In cases where it is not programmed by the manufacturer, it must be programmed with a unique EUI that you own.
+  link: https://www.thethingsnetwork.org/docs/lorawan/addressing.html
+
+Network:
+  description: The Network Server is responsible for routing Internet of Things data between devices and applications.
+  link: https://www.thethingsnetwork.org/docs/network/
+  alt: 
+    - Network Server
+    - Backend
+
+Application:
+  description: Applications are whatever your device communicates with on the internet. This could be as simple as an IFTTT Maker Applet or a visual flow using Node-RED to custom code on some server.
```

Comment:
```suggestion
  description: Applications are whatever your device communicates with on the internet. This could be as simple as an IFTTT Maker Applet or a visual flow using Node-RED to custom code on some server. It may also refer to an Application in the Application Server of The Things Stack in which LoRaWAN devices are registered. Applications can be created via the Console or CLI.


```